3.5-windowsservercore-1709/runtime/Dockerfile

```diff
@@ -1,11 +1,13 @@
+# escape=`
```

Comment:
This escape change seems incomplete.  Do these Dockerfiles compile?  I would expect the line continuation character to be changed on lines 5, 6, and 7. This comment applies to all of the Dockerfiles.